### PR TITLE
Add boards support for os-agent

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

File: boards/yellow/yellow.go [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	optLEDHeartbeat bool
+)
+
+type yellow struct {
+	conn  *dbus.Conn
+	props *prop.Properties
+}
+
+func getStatusLEDPower() bool {
+	value, _ := bootfile.ReadOption(bootConfig, "dtparam=pwr_led_trigger", "on")
```
Comment (agners) [inline boards/yellow/yellow.go:30]: The trigger is used as `linux,default-trigger`, `on` is not a valid trigger.

I think we need to check the reverse here: If it is `none`, then the LED is disabled.

Comment (pvizeli) [reply boards/yellow/yellow.go:30]: Well, that is just a soft default because not exists meaning its on

**Review by agners — APPROVED** [review]

File: utils/bootfile/editor.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		if strings.HasPrefix(line, optionName) {
+			outLines = append(outLines, "#"+line)
+		} else {
+			outLines = append(outLines, line)
+		}
+	}
+	file.Close()
+
+	// Write all lines back to boot config file
+	file, err = os.Create(filePath)
```
Comment (ties) [inline utils/bootfile/editor.go:58]: This does not look atomic. Probably low risk for this usage, but it is a smell. You might want to write a temporary file and `os.Rename` it into place.

Comment (agners) [reply utils/bootfile/editor.go:58]: Yeah good point. Given that its a boot file we should be careful about it. There seem to be Go libraries to write atomic, e.g. https://github.com/natefinch/atomic

Comment (agners) [reply utils/bootfile/editor.go:58]: Addressed with #99.

**Pull request merged**